### PR TITLE
Additional features

### PR DESCRIPTION
**Pull request by thorncorona**
(Review this 3rd after #7 )

Finish additional features 
- Delete comments
- Show a specified number of comments.

### CODE REVIEW

**Comment by thorncorona** [comment]
Okay so I've changed this branch to merge into `datastore` which is the branch it comes from. That fixed the issue with duplicated commits so you should be able to look at the relevant code. 

Once you approve the changes then I would need to rebase and merge into master, which would be a second PR right?

**Comment by arunk054** [comment]
No, you would not have to send another PR once the parent is merged. All you have to do is first, make sure it is in sync with latest changes in the `datastore` branch, then after you merge that, click edit and set `base:master`.  You could also go to your git client and set `git branch -u origin/master; git pull` which would automatically rebase your branch to the changes in master.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+ */
+@WebServlet("/delete-task")
+public class DeleteTaskServlet extends HttpServlet {
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    long id = Long.parseLong(request.getParameter("id"));
+    String page = request.getParameter("page");
+
+    System.out.println(id);
```
Comment (arunk054) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java:38]: Avoid these console print statements, if there is a logging library that you could use that would be preferable.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    System.out.println(page);
+
+    Key taskEntityKey = KeyFactory.createKey("Comment_" + page, id);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    datastore.delete(taskEntityKey);
```
Comment (arunk054) [inline portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java:43]: consider if you need to do any error checks  or exception handling here.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java (hunk 1 above)
Comment (thorncorona) [reply portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java:38]: Whoops yes accidentally left these debug statements in. I'll remove 'em.

File: portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java (hunk 2 above)
Comment (thorncorona) [reply portfolio/src/main/java/com/google/sps/servlets/DeleteTaskServlet.java:43]: I don't think I would right? 

There are two cases to consider: 
1) Input is valid, delete succeeds -> no error handling is needed
2) Input is invalid, delete fails -> this is either a result of the data already being deleted, or someone putting invalid data into the POST. In the first case, the comments refresh would make it look like the comment was deleted. In the second case, we know all data from the server is correct so if someone actively puts in bad data then we wouldn't need to create special responses for them. `datastore.delete` accepts arguments of bad keys.

**Pull request merged**